src/MasterPeace/Bundle/QuizBundle/Entity/QuizResult.php

```diff
@@ -25,20 +23,6 @@ class QuizResult
     private $id;
 
     /**
-     * @var Answer
-     *
-     * @ORM\ManyToOne(targetEntity="MasterPeace\Bundle\BookBundle\Entity\Answer")
-     */
-    private $answer;
-
-    /**
-     * @var Question
-     *
-     * @ORM\ManyToOne(targetEntity="MasterPeace\Bundle\BookBundle\Entity\Question")
-     */
-    private $question;
```

Comment:
Pabandžiau pataisyti. Dabartinėje mūsų schemoje quizResult to quizResulAnswers rysys yra one to many.